### ISSUE TITLE
ERROR 400: File is over the size limit"

### ISSUE THREAD

**Issue by TheRealVoodu** [bkeiren/EasyImgur#58]
So I was trying to upload a .gif file of 18MB using EasyImgur and it throws me an error 400 saying the file is too large. Please advice on how to fix this.

But when directly uploaded through the website imgur gives no such error for the same file. Its also clearly mentioned in their site at ( https://help.imgur.com/hc/en-us/articles/208606616-What-is-GIFV- ) that GIFs upto 200MB size are supported and will be converted to GIFv files.

-----------------------------------------------------------------------------------------------
Image uploaded directly to imgur site : https://imgur.com/pzXksMm

-----------------------------------------------------------------------------------------------
Please find below the log file and a screenshot of the image stats:

[easyimgur.log](https://github.com/bkeiren/EasyImgur/files/2453835/easyimgur.log)

![file stat](https://user-images.githubusercontent.com/43928017/46581153-dd12ee80-ca50-11e8-8ab7-f72ec5f833f6.png)

**Comment by bkeiren** [comment]
Hey @TheRealVoodu ,

Unfortunately the Imgur API that we programmatically talk to only allows images up to 10MB to be uploaded ([source](https://apidocs.imgur.com/#c85c9dfc-7487-4de2-9ecd-66f727cf3139)).

The likely reason why the website doesn't have this limit is because they can technically handle these sizes, but simply want to reserve this ability for themselves. The fact that there even is an API that third parties like me can work with is great, but we can't forget that Imgur isn't just an image hosting service but also social platform. As such, it benefits from users directly interacting with the website or its own social app(s) and conceivably limits the API in ways that drive traffic and users to the portal instead.

In short, there is unfortunately nothing I can do about this at this time. :(

Thanks for reporting this though, I hope it hasn't caused you too much trouble!
